### PR TITLE
Feature/104 mape calculator

### PR DESCRIPTION
**Pull request by Kristof-K**
## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos)
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Did you update the CHANGELOG

## PR review
Anyone in the community is free to review the PR once the tests have passed.

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones to the PR so it can be classified

### CODE REVIEW

**Comment by Kristof-K** [comment]
MAPE doesn't not log any warnings, if ground_truth 0 values exist. Should I implement that?

**Comment by benHeid** [comment]
Thank you for the Pull Request, can you rebase the branch on the origin master?

**Review by benHeid — CHANGES_REQUESTED** [review]
Thank you for the Pull Request, can you rebase the branch on the origin master?

**Comment by Kristof-K** [comment]
feature/104 mape calculator is rebased on current master

File: pywatts/summaries/mape_summary.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+                   Default 0
+    :type offset: int
+    :param filter_method: Filter which should performed on the data before calculating the MAPE.
+    :type filter_method: Callable[[np.ndarray, np.ndarray], Tuple[np.ndarray, np.ndarray]]
+    """
+
+
+    def _apply_metric(self, p, t):
+        non_zero_indices = t.nonzero()[0][t.nonzero()[0] >= self.offset]
+        return np.mean(np.abs((p[non_zero_indices] - t[non_zero_indices]) / t[non_zero_indices]))
```
Comment (benHeid) [inline pywatts/summaries/mape_summary.py:24]: I think you need to multiply the mean with 100. See [Mape wikipedia](https://en.wikipedia.org/wiki/Mean_absolute_percentage_error)

**Review by benHeid — CHANGES_REQUESTED** [review]

File: pywatts/summaries/mape_summary.py (hunk 1 above)
Comment (Kristof-K) [reply pywatts/summaries/mape_summary.py]: In my lecture "Forecasting" I got to know MAPE without the factor 100 (so I guess it is a conventional decision and in the book "Forecasting: Principles and Practice" from Hyndman and Athanasopoulos MAPE is also introduced with factor 100). Hence, I've added the factor 100.

**Review by benHeid — APPROVED** [review]

**Pull request merged**